src/assets/scss/delivery-options/style.scss

```diff
@@ -14,6 +14,14 @@
 
   position: relative;
 
+  .pb-2 {
```

Comment:
Waarom is dit niet geprefixed met $classBase? Bootstrap gebruikt !important dus dat heeft voorrang op deze class. Of is het de bedoeling om voor BS websites automatisch mee te sizen?

Reply:
Yes, dat is precies de bedoeling :)